apps/etterlatte-behandling/src/main/kotlin/behandling/klage/KlageService.kt

```diff
@@ -424,6 +433,32 @@ class KlageServiceImpl(
         )
     }
 
+    override fun fattVedtak(
+        klageId: UUID,
+        brukerTokenInfo: BrukerTokenInfo,
+    ) {
+        inTransaction {
+            val klage = klageDao.hentKlage(klageId) ?: throw NotFoundException("Klage med id=$klageId finnes ikke")
+
+            val vedtakId =
+                runBlocking {
+                    vedtakKlient.fattVedtakKlage(klage.id, brukerTokenInfo, klage.sak.enhet)
+                }
+            klageDao.lagreKlage(klage.copy(status = KlageStatus.FATTET_VEDTAK))
```

Comment:
```suggestion
            klageDao.lagreKlage(klage.fattVedtak())
```
og så ha status-endring i en funksjon på klage-klassen